pkgs/development/compilers/gmqcc/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, pkgs, ... }:
+pkgs.stdenv.mkDerivation rec {
+    pname = "gmqcc";
+    version = "297eab9";
```

Comment:
not a valid version, for intermediate commits you do something like  unstable-yyyy-mm-dd

idk right now the reference, people might comment here later